opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -738,4 +793,44 @@ class BrineCo2Pvt
 
 } // namespace Opm
 
+namespace Opm::gpuistl {
```

Comment:
Prefer brace on new line, and a blank line before the start of the function definition.